src/terra-dev-site/contributing/EntranceCriteria.a.contributing.mdx

```diff
@@ -0,0 +1,26 @@
+# Entrance Criteria
+
```

Comment:
We should move these sections from the pre-reqs page:
- prereqs intro paragraph
- https://engineering.cerner.com/terra-ui/contributing/terra-ui/prerequisites#project-governance
- https://engineering.cerner.com/terra-ui/contributing/terra-ui/prerequisites#we-welcome-prs-regarding

Then we can add a blur for medium or large contributions that are not included in the 'pr welcome list' to mention we accept larger contribution but require a stricter process because of the topics outlines here with UX Approval, Propietary concerns, etc.

Reply:
I think it makes more sense to re-name this section 'New component entrance criteria and to move the governance and we welcome Prs sections to the contributing doc

Reply:
Actually I've added the governance and code of conduct sections as their own pages on the getting started page